plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraTransform.kt

```diff
@@ -62,4 +64,35 @@ internal object CameraTransform {
   }
 
   fun calculateScaleBy(amount: Double, currentZoom: Double) = log2(amount) + currentZoom
+
+  /**
+   * Extension function to wrap coordinate to one world copy.
+   */
+  fun Point.wrapCoordinate(): Point {
+    val lng = wrap(
+      value = this.longitude(),
+      min = -LONGITUDE_MAX.toDouble(),
+      max = LONGITUDE_MAX.toDouble()
+    )
+    return if (lng.isNaN()) Point.fromLngLat(this.longitude(), this.latitude())
```

Comment:
Wonder in which case it can be NaN. Should `wrap(` just always return correct value?

Reply:
`wrap` could return NaN if arguments `min` and `max` are same, although that's not the case in our code. It's just an extra check so `wrapCoordinate` doesn't return NaN lat/lng.